src/Gui/DlgSettingsUnitsImp.cpp

```diff
@@ -38,19 +38,6 @@
 using namespace Gui::Dialog;
 using namespace Base;
 

```

Comment:
This block must not be deleted as otherwise it breaks the translation system.

Reply:
I thought this too and had a look. I could not see how our translation system still needs it. As test I removed the line

`qApp->translate("Gui::Dialog::DlgSettingsUnits", "FEM (mm, N, sec)");`

and run lupdate. Maybe I made a mistake but I did not see any difference. Therefore I passed the removal.

How is lupdate evaluating this code chunk?

Reply:
lupdate looks for calls of "translate" independent whether it's inside an `#if 0` block or regular code.